Automattic/jetpack
# Sitemaps: align with Google deprecations

Fixes #24278

#### Changes proposed in this Pull Request:

* Removes image sitemap fields no longer used.

#### Jetpack product discussion


n/a See https://developers.google.com/search/blog/2022/05/spring-cleaning-sitemap-extensions

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* On a Jetpack connected site with Sitemaps enabled, add images. some images.
* Verify they exist on the image sitemap.
* Run it through Google's sitemap tool ( https://support.google.com/webmasters/answer/7451001 )
* Update to this patch, repeat and confirm it passes still.
